_harp/academy/_faq.md

```diff
@@ -22,7 +22,10 @@ Courses are peer-led and project-based with sixteen students working in four tea
 Primarily, we cover *test-driven* development, using *JavaScript* and *Node.js*. We also cover aspects of UX design, product management and entrepreneurship.
 
 **What projects will I work on in the second part of the programme?**  
-We aim to give you an opportunity to work on a variety of open source, charitable, social enterprise, startup and commercial projects. The programme is under continuous development and the balance between learning and earning a living is always under review.  
+We aim to give you an opportunity to work on a variety of open source, charitable and social enterprise projects. The programme is under continuous development and the balance between learning and creating useful software is always under review.  
+
+**Will I be able to earn a living working on projects?**  
```

Comment:
@iteles see https://github.com/foundersandcoders/business/issues/12. My intention is to fill everybody's schedule with *unpaid* charity/social/self work for the final 4 weeks of the 16. Hence the change of wording from *earning a living* to *creating useful software*. I am not expecting this pull request to be merged until after the issue above has been discussed at our meeting on the 2nd.